.github/workflows/csi_operator_ci.yaml

```diff
@@ -0,0 +1,84 @@
+name: Community Operators test
+on:
+  push:
+    branches:
+      - develop
+      - master
+  pull_request:
+    branches:
+      - develop
+      - master
+jobs:
+  operator_image_build:
+   name: "set env"
+   runs-on: ubuntu-latest
+   steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+   outputs:
+     operator_image_for_test: "ibmcom/ibm-block-csi-operator:latest"
+
+  prepare_env:
+   name: "prepare env"
+   needs: operator_image_build
+   runs-on: ubuntu-latest
+   steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+      - name: Install dependencies
+        run: python -m pip install --upgrade pip yq
+      - name: install apt dependencies
+        run: |
+          build/ci/install_ci_dependencies.sh
```

Comment:
`install` -> `Install`
why are these even separated?
if there's a good reason, then at least consider renaming the 1st to something more indicative, depending on the reason

Reply:
done